shell/common/shell_test_platform_view.cc

```diff
@@ -32,28 +21,18 @@ std::unique_ptr<ShellTestPlatformView> ShellTestPlatformView::Create(
   // Make this fully runtime configurable
   switch (backend) {
     case BackendType::kDefaultBackend:
```

Comment:
Ugh. Worked out the issue. This is relying on falling through to a backend that is not `#ifdef`'ed out.

I'll kill this enum value and replace it with a function that returns the appropriate actual backend rather that using this trick in every single switch statement on this enum.

Reply:
Sent https://github.com/flutter/engine/pull/56744 for that cleanup.